main.lua

```diff
@@ -88,6 +88,9 @@ function Initialize(Plugin)
 		cFile:CreateFolder(homeDir)
 	end
 
+	--Dont know where to specify this
+	SocialSpyList = {}
```

Comment:
Yes, it should be placed in main.lua --Global variables-- at the beggining of file